service/resource/configmap/desired.go

```diff
@@ -14,6 +15,7 @@ import (
 )
 
 func (r *Resource) GetDesiredState(ctx context.Context, obj interface{}) (interface{}, error) {
+	r.logger.Log("debug", fmt.Sprintf("fetching configmap: %s/%s", r.configMapNamespace, r.configMapName))
```

Comment:
The extra logging helped me track down the issue, I reckon we might as well keep it.